scopesim/commands/scopesimple.py

```diff
@@ -268,10 +269,11 @@ def plot(self, img_slice=None, adjust_scale=False, **kwargs):
                 " create an uneven image plot.")
 
         vminmax = self._get_vminmax(adjust_scale)
+        fig_kwargs = kwargs.pop("fig_kwargs", {})
```

Comment:
I'm going to suggest this anyway: I think it would be better to just add `fig_kwargs` as an optional parameter to the function instead of popping it from `kwargs`. As far as I'm aware, there is no good argument for doing it this way; or am I missing something?

Reply:
Yeah I did that first, but then there would be `fig_kwargs` followed by "normal" `**kwargs` (which go into the plot but not the figure), and also we'd need a default, but dict is mutable so need to use None, then need another line `if fig_kwargs is None` and so on. But maybe I should have done that rather than this intransparent implicit kwarg-popping. I'll probably change at some point...

Reply:
Yes